packages/tools/src/types/ToolSpecificAnnotationTypes.ts

```diff
@@ -260,18 +260,25 @@ export type PlanarFreehandROIAnnotation = ContourAnnotation & {
 export type PlanarFreehandContourSegmentationAnnotation =
   PlanarFreehandROIAnnotation & ContourSegmentationAnnotationData;
 
-export type InterpolationROIAnnotation = ContourAnnotation & {
-  metadata: {
-    annotationUID?: string;
+export type InterpolationROIAnnotation = ContourAnnotation &
+  ContourSegmentationAnnotationData & {
+    metadata: {
+      annotationUID?: string;
```

Comment:
I know you didn't change this now but do you know the difference between `annotation.annotationUID` from `Annotation` and `annotation.metadata.annotationUID` from `InterpolationROIAnnotation?

Reply:
No idea